### PR TITLE
Vet clinic database: database performance audit

### PR DESCRIPTION
**Pull request by ZewdieMc**
### In this PR, 

- [x]  Added email as extra column to the owners table.

- [x]  Inserted given data.

## Quries 
-  `SELECT COUNT(*) FROM visits where animal_id = 4;`
### Before:
1before

### After:

-  `SELECT * FROM visits where vet_id = 2;`
### Before:
2before

### After:

-  `SELECT * FROM owners where email = '[owner_18327@mail.com](mailto:owner_18327@mail.com)';`
### Before:
3before

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -171,4 +171,10 @@ JOIN(SELECT ANIMALS.SPECIES_ID, COUNT(*) AS SP_COUNT FROM ANIMALS
...
 	WHERE VETS.NAME='Maisy Smith' GROUP BY ANIMALS.SPECIES_ID
 	ORDER BY SP_COUNT DESC LIMIT 1) SP
-ON SP.SPECIES_ID=SPECIES.ID;
\ No newline at end of file
+ON SP.SPECIES_ID=SPECIES.ID;
+
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animal_id = 4; 
+
+EXPLAIN ANALYZE SELECT * FROM visits where vet_id = 2;
+
```
Comment (thecodechaser) [inline queries.sql:179]: - [ ] In the screenshots for the first two query results the execution time is bigger than 1000ms but it should be smaller then 1000ms as requested in the project requirements. Please make sure to fix this and update the screenshot in the PR description. You can follow the below steps to achieve this.

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @ZewdieMc, @torobucci, and @danutibe07👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (ZewdieMc) [reply queries.sql:179]: Hi @thecodechaser, 
Thank you for your feedback.
Kindly note that the point is not among the project requirements, it's a preparation to proceed to the project requirements and it's saying that the execution time of the `SELECT count(*) from visits where animal_id = 4` should be `>= 1000ms` not `< 1000ms`.
We apologize if we misunderstood it.

**Review by addod19 — APPROVED** [review]
### Hi @ZewdieMc , @torobucci , @danutibe07, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [ ] The execution time for the before queries are higher than 1000ms :100: 
- [ ] Index scan shows on the query :rocket: 
- [ ] Explain analyze used to show the details of the query results :fireworks: 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ZewdieMc** [comment]
@addod19 
Thank you, we are grateful.

**Pull request merged**